plugin/src/main/java/org/opennms/plugins/cloud/config/PrerequisiteChecker.java

```diff
@@ -0,0 +1,24 @@
+package org.opennms.plugins.cloud.config;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PrerequisiteChecker {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PrerequisiteChecker.class);
+
+    private PrerequisiteChecker() {
+        // Utility class
+    }
+
+    static boolean isSystemIdOk(final String systemId) {
+        if(systemId == null ||
+                systemId.length() < 36 ||
+                systemId.matches("[0-]*")){
```

Comment:
How frequent it will isSystemIdOk called ? If it calling once per few hours it is fine. Otherwise suggest to pre compile the regex.